lms/services/lti_outcomes.py

```diff
@@ -98,17 +98,12 @@ def _send_request(self, request_body):
         # Bind the variable so we can refer to it in the catch
         response = None
 
-        try:
-            response = self.http_service.post(
-                url=self.service_url,
-                data=xml_body,
-                headers={"Content-Type": "application/xml"},
-                auth=self.oauth1_service.get_client(),
-            )
-        except HTTPError as err:
-            raise ExternalRequestError(
-                "Error calling LTI Outcomes service", response
-            ) from err
```

Comment:
`LTIOutcomesClient` was catchin `HTTPError` and re-raising it as `ExternalRequestError`. Now that `HTTPService` raises `ExternalRequestError` directly `LTIOutcomesClient` no longer needs a `try` / `except` at all here. (This does mean that the `"Error calling LTI Outcomes service"` string has been lost but I don't think that was adding anything.)

(`LTIOutcomesClient` previously caught `HTTPError` and raised `LTIOutcomesAPIError`, a pointless `ExternalRequestError` subclass, but that was recently replaced with just raising `ExternalRequestError` directly.)